andela/bad_ass-ah-frontend
# #164489906 Highlight a particular text in article body

**What does this PR do?**

- Highlight a section of an article body

**Description of Task to be completed?**

- Giving the user the ability to mark or highlight a particular part of the article

**How should this be manually tested?**

- Navigate to a particular article
- Select a particular part and you will see a popup with a highlight button, then click that button
- You will get a text box to add your comment and then send, and your text  will be highlighted

**What are the relevant pivotal tracker stories?**

[#164489906](https://www.pivotaltracker.com/story/show/164489906)

**Screenshots (if appropriate)**
<img width="655" alt="Screen Shot 2019-06-10 at 09 26 25" src="https://user-images.githubusercontent.com/32579116/59179687-2462f900-8b63-11e9-9158-16f3ff215cc9.png">
<img width="589" alt="Screen Shot 2019-06-10 at 09 29 51" src="https://user-images.githubusercontent.com/32579116/59179696-2b8a0700-8b63-11e9-89ac-0ec8ec2950cb.png">
